code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -125,12 +130,17 @@
 	new /obj/item/clothing/glasses/sunglasses/sechud(src)
 	new /obj/item/clothing/glasses/sunglasses/sechud/head(src)
 	new /obj/item/shield/riot/tact(src)
+	new /obj/item/device/flashlight/maglight(src)
 	new /obj/item/melee/telebaton(src)
 	new /obj/item/gun/energy/repeater/pistol(src)
 	new /obj/item/taperoll/police(src)
 	new /obj/item/device/breath_analyzer(src)
 	new /obj/item/crowbar/red(src)
 	new /obj/item/ipc_tag_scanner(src)
+	new /obj/item/device/taperecorder(src)
+	new /obj/item/device/binoculars(src)
+	new /obj/item/device/radio/sec(src)
+	new /obj/item/device/multitool(src)
```

Comment:
Why does the hos need a multi tool?

Reply:
it was already there, it's for locking/unlocking suit sensors on prisoner uniforms